engines_test.go

```diff
@@ -28,7 +40,11 @@ func TestListEngines(t *testing.T) {
 	client, server, teardown := setupOpenAITestServer()
 	defer teardown()
 	server.RegisterHandler("/v1/engines", func(w http.ResponseWriter, r *http.Request) {
-		resBytes, _ := json.Marshal(EnginesList{})
+		engines := make([]Engine, 5)
```

Comment:
I guess we can use random int here instead

Reply:
Done